cmd/kwild/config/flags.go

```diff
@@ -24,6 +24,8 @@ func AddConfigFlags(flagSet *pflag.FlagSet, cfg *KwildConfig) {
 	flagSet.StringVar(&cfg.AppCfg.TLSCertFile, "app.tls-cert-file", cfg.AppCfg.TLSCertFile, "TLS certificate file path for the admin and consensus RPC server (optional)")
 	flagSet.StringVar(&cfg.AppCfg.TLSKeyFile, "app.tls-key-file", cfg.AppCfg.TLSKeyFile, "TLS key file path for the admin and consensus RPC servers (optional)")
 	flagSet.StringVar(&cfg.AppCfg.Hostname, "app.hostname", cfg.AppCfg.Hostname, "kwild Server hostname")
+	flagSet.BoolVar(&cfg.AppCfg.AutheticateRPCs, "app.authenticate-rpcs", cfg.AppCfg.AutheticateRPCs, "Authenticate JSON-RPC requests using the private key")
```

Comment:
Instead of `authenticate-rpcs`, maybe do `private-rpc`? Simply because we only authenticate one of the RPC methods (`call`), we block another (`query`), and modify but don't authenticate the contents of a third (`tx status`).

Reply:
Will make this change to `private-rpc`.  I agree with your points, just missed this comment.